src/components/Canvas/Editor.tsx

```diff
@@ -3007,6 +3031,12 @@ export default class Editor extends EventDispatcher {
       this.undo();
     } else if (e.code === "KeyY" && (e.ctrlKey || e.metaKey)) {
       this.redo();
+    } else if (e.code === "ShiftLeft" || e.code === "ShiftRight") {
+      if (!this.isShiftPressed && this.brushTool === BrushTool.DOT) {
+        this.isShiftPressed = true;
+        this.originalBrushTool = this.brushTool;
+        this.setBrushTool(BrushTool.LINE);
+      }
```

Comment:
Interesting that you enabled line brush tool when pressing Shift key. Is there a particular reason to this?

Reply:
Also, it would be great if you could add in the documentation that pressing shift key allows activation of line tool! Some users might want to disable the pressing shift key activating line tool. But I believe unless there is nobody wanting to manipulate the shift key press themselves, I think we can just go with your written code.

Reply:
no particular reason, just added it for convenience from personal experience, i can remove it if you don't think it fits the codebase, not that important for me

Reply:
I think removing it will be better for now. Developers can implement such behavior using hooks I believe!🙏

Reply:
here you go, i removed it